tools/project_manager/show_good_word_suggestions_detail.php

```diff
@@ -49,8 +49,6 @@
 $details = json_encode([
     "layout" => $layout,
```

Comment:
It would be nice to be able to switch layout without reloading the page. But this code saves layout changes in user settings. To do this we would have to implement an api for GET and PUT to user settings. Do we want to do this or would it be better to save changes locally instead?